### PR TITLE
Admin can edit photos

### PR DESCRIPTION
**Pull request by chadellison**

### CODE REVIEW

**Comment by chadellison** [comment]
Allow Admin to edit photos
Add platform admin to seed file: username: "jorge@turing.io", password: "password"

File: app/controllers/platform_admin/photos_controller.rb [hunk 1]
```diff
@@ -3,10 +3,33 @@ def index
...
 
+  def edit
+    @photo = Photo.find(params[:id])
+  end
+
+  def update
+    photo = Photo.find(params[:id])
+    # photo.update(photo_params)
+    vendor = Vendor.find(photo.vendor.id)
+    vendor.photos.find(photo.id).update(photo_params)
```
Comment (s-espinosa) [inline app/controllers/platform_admin/photos_controller.rb]: I think if we're not updating the vendor here we should be able to just directly update the photo. What's wrong with the commented out line 12? Think we need vendor.photos.create when we create the photo to establish the relationship, but now that we have that, should be able to directly edit the photo. Recommend using line 12 and deleting 13 and 14. If there's a reason not to that I'm missing, recommend deleting line 12.

File: db/seeds.rb [hunk 2]
```diff
@@ -15,6 +16,15 @@ def create_categories
     puts "Created Categories"
   end
 
+  def platform_admin
+    user = User.create(username: "jorge@turing.io",
```
Comment (s-espinosa) [inline db/seeds.rb]: Let's make the plat_admin username just "jorge".

**Comment by s-espinosa** [comment]
@chadellison Looks like there are conflicts with master that need to be resolved. Also provided some comments on the a couple of lines in the files. Recommend that you:

* Address the comments
* Squash your commits
* Checkout master/pull master
* Checkout your branch/merge master into your branch/resolve any merge conflicts

Let me know what you think!

**Pull request merged**